From b7a1df23b82bf5e3aca048822f79a1d5aab3e32d Mon Sep 17 00:00:00 2001
From: Vendula Poncova <vponcova@redhat.com>
Date: Tue, 27 Aug 2019 13:36:35 +0200
Subject: [PATCH] Don't set up a warning label to None (#1745933)

If the error is None, set the warning label to an empty string.

Resolves: rhbz#1745933
---
 pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py b/pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py
index 0c6b391cf6..62caeaa236 100644
--- a/pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py
+++ b/pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py
@@ -277,7 +277,7 @@ def on_add_confirm_clicked(self, button, *args):
         else:
             self._error = validate_mount_point(self.mount_point, self.mount_points)
 
-        self._warningLabel.set_text(self._error)
+        self._warningLabel.set_text(self._error or "")
         self.window.show_all()
         if self._error:
             return
